src/lgar.cxx

```diff
@@ -2335,15 +2527,16 @@ extern void lgar_dzdt_calc(bool use_closed_form_G, int nint, double h_p, int *so
    is within a tolerance. */
 // ############################################################################################
 extern double lgar_theta_mass_balance(int layer_num, int soil_num, double psi_cm, double new_mass,
-				      double prior_mass, double *delta_theta, double *delta_thickness,
-				      int *soil_type, struct soil_properties_ *soil_properties)
+				      double prior_mass, double *AET_demand_cm, double *delta_theta, double *delta_thickness,
+				      int *soil_type, struct soil_properties_ *soil_properties, struct wetting_front** head)
```

Comment:
You're correct. I just included head because it's a necessary parameter for printing, which I included in this fxn as part of debugging. Now removing head from this fxn.